### ISSUE TITLE
typo of discriminator loss in non-saturating type

### ISSUE THREAD

**Issue by XiaoluDing** [SerezD/vqvae-vqgan-pytorch-lightning#3]
https://github.com/SerezD/vqvae-vqgan-pytorch-lightning/blob/master/vqvae/modules/loss/loss.py#L38C14-L38C14
        fake_loss = functional.binary_cross_entropy_with_logits(logits_fake, target=torch.ones_like(logits_fake))

maybe fix it using zeros_like

**Comment by SerezD** [comment]
Hi, thank you for pointing that out. 
The corrected version is now online.
